draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -784,4 +798,73 @@ private void notifyCloseToLeftListener() {
   public int getDraggedViewHeightPlusMarginTop() {
     return transformer.getMinHeightPlusMargin();
   }
+
+  public void initializeTimers(){
+
+    onMaximizeCountdownTimer = new CountDownTimer(2000, 10) {
+      @Override
+      public void onTick(long l) {
+
+        if (getSecondViewAlpha() > 0.95) {
+          notifyMaximizeToListener();
+          new Handler().post(new Runnable() {
+            @Override
+            public void run() {
+              onMaximizeCountdownTimer.cancel();
+            }
+          });
+        }
+      }
+
+      @Override
+      public void onFinish() {
+
+      }
+    };
```

Comment:
You are right. Since the alpha is modified constantly in that case, removing the extra timers and handling the notifiers within the alpha change sounds logical. I'll be implementing and testing that version. Thanks!